osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -184,6 +184,12 @@ public ScoreInfo DeepClone()
         [Ignored]
         public bool IsLegacyScore => Mods.OfType<ModClassic>().Any();
 
+        /// <summary>
+        /// Score multiplier of this score. Value will be correct only if <see cref="mods"/> is non-null.
+        /// </summary>
+        [Ignored]
+        public double ScoreMultiplier { get; private set; } = 1;
```

Comment:
If you have to preempt that a property is only valid to read in specific circumstances, either enforce that the circumstances are met via an exception or something, or, preferably, just don't add it at all, and inline it in the specific circumstance you need to use it in.